release notes/upcoming.md

```diff
@@ -47,6 +47,11 @@ Thanks to @danron for their work on this!
 
 Previously the `setup()` and `teardown()` functions timed out after 10 seconds. Now that period is configurable via the `setupTimeout` and `teardownTimeout` script options or the `K6_SETUP_TIMEOUT` and `K6_TEARDOWN_TIMEOUT` environment variables. The default timeouts are still 10 seconds and at this time there are no CLI options for changing them to avoid clutter.
 
+### In-client aggregation for metrics streamed to the cloud (#600)
+
+Metrics streamed to the LoadImpact cloud can be partially aggregated to reduce bandwidth usage and processing times. Outlier metrics are automatically detected and excluded from that aggregation.
```

Comment:
"LoadImpact" -> "Load Impact". This is somewhat of a discussion internally if we should officially switch, but right now the correct is "Load Impact".